ramalama/model.py

```diff
@@ -128,14 +128,10 @@ def quadlet(self, args, exec_args):
 [Container]
 Device=+/dev/dri
 Device=+/dev/kfd
-Environment=RAMALAMA_TRANSPORT=%s
 Exec=%s
 Image=%s
-Label=RAMALAMA container
-%s
 SecurityLabelDisable=true
-Volume=%s:/usr/bin/ramalama/ramalama:ro
-Volume=%s:/var/lib/ramalama:ro
```

Comment:
Without bind mounting /var/lib/ramalama the container won't have access to models, in the rootless case we do something like:

```
-v/var/home/ecurtin/.local/share/ramalama:/var/lib/ramalama
```

Reply:
Good point.  We should just volume mount in the image